service/history/api/consistency_checker.go

```diff
@@ -172,7 +172,7 @@ func (c *WorkflowConsistencyCheckerImpl) getWorkflowContextValidatedByClock(
 		}
 	}
 
-	wfContext, release, err := c.workflowCache.GetOrCreateWorkflowExecution(
+	workflowLease, release, err := c.workflowCache.GetOrCreateWorkflowExecution(
```

Comment:
I thought we are renaming api.WorkflowContext/Lease object. This one from workflowCache is workflow.Context?

Reply:
I renamed the function that used to say Context as well, now - to make that clearer, too